src/schemes/boundary/system.jl

```diff
@@ -12,49 +13,56 @@ The interaction between fluid and boundary particles is specified by the boundar
 - `movement`: For moving boundaries, a [`BoundaryMovement`](@ref) can be passed.
 - `adhesion_coefficient`: Coefficient specifying the adhesion of a fluid to the surface.
    Note: currently it is assumed that all fluids have the same adhesion coefficient.
+- `viscosity`:  Free-slip (default) or no-slip condition. See description above for further
+                information.
```

Comment:
```suggestion
- `viscosity`:  Free-slip (default) or no-slip condition.
```
There is no further information in the description. I'm wondering where this comes from, as the models (where you copied this from) also don't contain any additional information in the description.